app/presenters/order_detail_presenter.rb

```diff
@@ -21,6 +21,10 @@ def description_as_html
     end.join(" &mdash; ").html_safe
   end
 
+  def description_as_html_with_facility_prefix
+    "#{facility.abbreviation} / #{description_as_html}".html_safe
```

Comment:
We'd have to go back and rethink `description_as_html`, which does its own escaping. Without the `html_safe` here, the `description_as_html` part gets displayed double-encoded when rendered.